Userland/Libraries/LibWeb/Cookie/ParsedCookie.cpp

```diff
@@ -222,6 +225,25 @@ void on_http_only_attribute(ParsedCookie& parsed_cookie)
     parsed_cookie.http_only_attribute_present = true;
 }
 
+void on_same_site_attribute(ParsedCookie& parsed_cookie, StringView attribute_value)
+{
+    // https://httpwg.org/http-extensions/draft-ietf-httpbis-rfc6265bis.html#name-the-samesite-attribute-2
+
+    // 1. Let enforcement be "Default"
+    // Note: Set as default value in ParsedCookie.h
+
+    if (attribute_value.equals_ignoring_case("None"sv)) {
+        // 2. If cookie-av's attribute-value is a case-insensitive match for "None", set enforcement to "None".
```

Comment:
These all go before their individual if statements

Reply:
I followed the convention from other speced functions in this file so did it like this
will make the change

did you mean something like this?
```
    // 2. If cookie-av's attribute-value is a case-insensitive match for "None", set enforcement to "None".
    if (attribute_value.equals_ignoring_case("None"sv)) {
        parsed_cookie.same_site_attribute = "None"sv;
        // 3 If cookie-av's attribute-value is a case-insensitive match for "Strict", set enforcement to "Strict".
    } else if (attribute_value.equals_ignoring_case("Strict"sv)) {
        parsed_cookie.same_site_attribute = "Strict"sv;
        // 4. If cookie-av's attribute-value is a case-insensitive match for "Lax", set enforcement to "Lax".
    } else if (attribute_value.equals_ignoring_case("Lax"sv)) {
        parsed_cookie.same_site_attribute = "Lax"sv;
    }
```
clang-format indents these else if comments
Or you meant this?
```
   // 2. If cookie-av's attribute-value is a case-insensitive match for "None", set enforcement to "None".
    if (attribute_value.equals_ignoring_case("None"sv))
        parsed_cookie.same_site_attribute = "None"sv;
    // 3 If cookie-av's attribute-value is a case-insensitive match for "Strict", set enforcement to "Strict".
    if (attribute_value.equals_ignoring_case("Strict"sv))
        parsed_cookie.same_site_attribute = "Strict"sv;
    // 4. If cookie-av's attribute-value is a case-insensitive match for "Lax", set enforcement to "Lax".
    if (attribute_value.equals_ignoring_case("Lax"sv))
        parsed_cookie.same_site_attribute = "Lax"sv;
```

Reply:
With else if we use the (admittedly a bit strange) format:
```cpp
    // 2. If cookie-av's attribute-value is a case-insensitive match for "None", set enforcement to "None".
    if (attribute_value.equals_ignoring_case("None"sv)) {
        parsed_cookie.same_site_attribute = "None"sv;
    } 
	// 3 If cookie-av's attribute-value is a case-insensitive match for "Strict", set enforcement to "Strict".
	else if (attribute_value.equals_ignoring_case("Strict"sv)) {
        parsed_cookie.same_site_attribute = "Strict"sv;
    } 
	// 4. If cookie-av's attribute-value is a case-insensitive match for "Lax", set enforcement to "Lax".
	else if (attribute_value.equals_ignoring_case("Lax"sv)) {
        parsed_cookie.same_site_attribute = "Lax"sv;
    }
```

But your second version seems fine as well

Reply:
In fact you might not even need the brackets, since each if body is just a single line

Reply:
yeah I tried it like this
```
    // 2. If cookie-av's attribute-value is a case-insensitive match for "None", set enforcement to "None".
    if (attribute_value.equals_ignoring_case("None"sv))
        parsed_cookie.same_site_attribute = "None"sv;
    // 3 If cookie-av's attribute-value is a case-insensitive match for "Strict", set enforcement to "Strict".
    else if (attribute_value.equals_ignoring_case("Strict"sv))
        parsed_cookie.same_site_attribute = "Strict"sv;
    // 4. If cookie-av's attribute-value is a case-insensitive match for "Lax", set enforcement to "Lax".
    else if (attribute_value.equals_ignoring_case("Lax"sv))
        parsed_cookie.same_site_attribute = "Lax"sv;
```
doesn't look like clang-format complains